src/actions.js

```diff
@@ -305,6 +305,14 @@ export function fetchAllRegions() {
   return graphql(payload, `LOCATION_REGIONS`);
 }
 
+export function fetchAllDistricts() {
+  let filters = [`type: "D"`, `name_Ne : "Funding"`];
```

Comment:
```suggestion
  let filters = [`type: "D"`, `name :{ ne: "Funding"}`];
```
NOT 100 % sure but it is wass is suggested here
see https://stackoverflow.com/a/42485077/3533113

Reply:
Because Django ORM doesn't have ne by default. Therefore, we have to create it.